docs/methods/cardanoGetAddress.md

```diff
@@ -18,6 +18,7 @@ TrezorConnect.cardanoGetAddress(params).then(function(result) {
 #### Exporting single address
 * `path` — *obligatory* `string | Array<number>` minimum length is `5`. [read more](path.md)
 * `address` — *optional* `string` address for validation (read `Handle button request` section below)
+* `protocolMagic` - *obligatory* `Integer` 0 for Mainnet, 42 for Testnet
```

Comment:
shouldn't that be a different number for mainnet?

Reply:
We don't know what the number will be. I asked IOHK on slack and we'll see. I will also mention it in the PR description.